language/documentation/book/src/modules-and-scripts.md

```diff
@@ -71,20 +68,54 @@ module Test {
         Debug::print(&sum)
     }
 }
-}
 ```
 
-The `address 0x42` part specifies that the module will be published under the [account address](./address.md) 0x42 in [global storage](./global-storage-structure.md).
+The `module 0x42::..` part specifies that the module `Test` will be published under the [account address](./address.md) `0x42` in [global storage](./global-storage-structure.md).
 
-Multiple modules can be declared in a single `address` block:
+Modules can also be declared using [named addresses](./address.md). For example:
 
 ```move
-address 0x42 {
-module M { ... }
-module N { ... }
+module TestAddr::Test {
+    struct Example has copy, drop { a: address}
+
+    use Std::Debug;
+    friend TestAddr::AnotherTest;
+
+    public fun print() {
+        let example = Example { a: @TestAddr};
+        Debug::print(&example)
+    }
 }
 ```
 
+Named addresses only exist at the source language level and during compilation
```

Comment:
See my comments on a previous instance of this text. Maybe we should also try to avoid saying the exact same thing in two places?

Reply:
Updated it to be more concise and not say the same thing. Let me know if the updated text looks good.